taskcluster/scriptworker_taskgraph/parameters.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.parameters import extend_parameters_schema
+from voluptuous import (
+    Any,
+    Optional,
+    Required,
+)
+
+PROJECT_SPECIFIC_PREFIXES = {
+    "refs/heads/dev-": "dev",
+    "refs/heads/production-": "production",
+}
+
+PUSH_TAGS = ("dev", "production")
+
+scriptworker_schema = {
+    Optional('docker_tag'): Any(basestring, None),
+    Optional('push_docker_image'): Any(True, "force", False, None),
```

Comment:
Nit: `False` is never used. I wonder if we should default to it, instead of `None`. That might help making allowed values more explicit.

Reply:
Sure. I couldn't find how to set default values on extended schemas; do you have an example?

Reply:
We don't support defaults in schemas. This comes from m-c, where `--fast` disables schema validation, which implements implements defaults by returning the new structure. Due to that, we ignore the return value, to make sure `--fast` doesn't change behavior.